develop/items/food.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Food Items
+description: Learn how to add a FoodComponent to an item to make it edible, and configure it.
+authors:
+  - IMB11
+---
+
+# Food Items
+
+Food is a core aspect of survival Minecraft, so when creating edible items you have to consider the food's usage with other edible items.
+
+Unless you're making a mod with overpowered items, you should consider:
+
+- How much hunger your edible item adds or removes.
+- What potion effect(s) does it grant?
+- Is it early-game or endgame accessible?
+
+## Adding the Food Component
```

Comment:
Shame, this all changes in newer versions, we should really update ASAP

Reply:
I will be updating everything to 1.21 after the block and blockstate pages are migrated.